author/how-to/fifthtry-hosting.ftd

```diff
@@ -0,0 +1,80 @@
+-- ds.page: Getting Started with FifthTry Hosting
+
+Welcome to FifthTry Hosting – the perfect hosting solution for `fastn` 
+enthusiasts. Follow these simple steps to have your `fastn-powered website`
+up and running in no time. 
```

Comment:
This is completely wrong. You are writing as fastn. We do not say "welcome to Vercel" hosting. Also remove the adjectives like perfect. This is not marketing copy, this is how to document.